public/locations-list-map.js

```diff
@@ -726,11 +730,11 @@ function getMarkers(data, appliedFilters, bounds, markerOptions) {
         let marker = entry.marker;
 
         if (marker) {
-          if (!inStateFilter || !inAcceptFilter) {
+          if (!inAcceptFilter) {
```

Comment:
Logic here is a little weird if there's an "accepts items" filter set. We'll filter the in-state markers (those that don't match the "accepts items" filter will be hidden). But we'll leave the out-of-state markers alone. That's consistent with the show-others features - pins for all out-of-country entries are all shown.

Reply:
Ah, this weird logic breaks behavior when no state filter is set. All items are then considered 'out of state' and the "accepts items" filter has no effect.